File: Data Structures & Algorithms/products-of-array-discluding-self/submission-1.cpp
class Solution {
public:
    vector<int> productExceptSelf(vector<int>& nums) {
        vector<int> out(nums.size(), 0);
        int id = 0;
        for(int i = 0; i<nums.size(); ++i){
            int res = 1;
            for(int j = 0; j<nums.size(); ++j){
    
                if(j!=i){
                    res *= nums[j];
                }
            }
            out[i]=res;
            id++;
        }
    return out;
    }

};

File: Data Structures & Algorithms/partition-equal-subset-sum/submission-1.py
class Solution:
    def canPartition(self, nums: List[int]) -> bool:
        if sum(nums) % 2 == 1:
            return False
        res = sum(nums) // 2
        li = set()
        li.add(0)
        for i in nums:
            newli = set()
            for j in li:
                newli.add(j+i)
                newli.add(j)
            li = newli
        if res in li:
            return True
        return False